Recast/Source/RecastLayers.cpp

```diff
@@ -27,7 +27,8 @@
 #include "RecastAssert.h"
 
 
-static const int RC_MAX_LAYERS = RC_NOT_CONNECTED;
+// Must be 255 or smaller
+static const int RC_MAX_LAYERS = 63;
```

Comment:
If `RC_MAX_LAYERS` is 255, then the max index will be 254, which should be fine I believe.